File: app/src/main/java/com/sky/chowder/api/view/IMainView.java
package com.sky.chowder.api.view;

import com.sky.api.IBaseView;
import com.sky.chowder.model.ActivityModel;

import java.util.List;

/**
 * Created by SKY on 2017/5/29.
 */
public interface IMainView extends IBaseView {

    void setData(List<ActivityModel> data);

}

File: app/src/main/java/com/sky/chowder/api/view/IMainView.kt
package com.sky.chowder.api.view

import com.sky.api.IBaseView
import com.sky.chowder.model.ActivityModel

/**
 * Created by SKY on 2017/5/29.
 */
interface IMainView : IBaseView {
    fun setData(data: List<ActivityModel>)
}